### PR TITLE
chore: use test dependencies from node_modules

### PR DESCRIPTION
**Pull request by DanielRuf**
This replaces https://github.com/zurb/foundation-sites/pull/11177 and updates the browser-based test file to use the dependencies from node_modules.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "babel-preset-env": "^1.6.1",
     "browser-sync": "^2.18.12",
     "browserstack-runner": "^0.7.0",
+    "chai": "^4.1.2",
```
Comment (ncoden) [inline package.json]: We must update both `package-lock.json` and `yarn.lock`. Please do the following:
```bash
rm package-lock.json
yarn
npm i --package-lock-only
sed -i '' 's/http\:\/\//https\:\/\//g' package-lock.json
sed -i '' 's/http\:\/\//https\:\/\//g' yarn.lock
```

File: test/javascript/index.html [hunk 2]
```diff
@@ -6,16 +6,16 @@
...
+    <link rel="stylesheet" href="../../node_modules/mocha/mocha.css" />
     <link rel="stylesheet" href="../../_build/assets/css/foundation-float.css">
     <link rel="stylesheet" href="../../node_modules/motion-ui/dist/motion-ui.css">
   </head>
   <body>
     <div id="mocha"></div>
-    <script src="lib/chai.js"></script>
-    <script src="lib/chai-jquery.js"></script>
-    <script src="lib/sinon.js"></script>
+    <script src="../../node_modules/chai/chai.js"></script>
```
Comment (ncoden) [inline test/javascript/index.html]: We use `chai` directly, so please install `chai` in devDependencies.

**Review by ncoden — CHANGES_REQUESTED** [review]

File: test/javascript/index.html (hunk 2 above)
Comment (DanielRuf) [reply test/javascript/index.html]: That's what I did (see the package.json)?

File: package.json (hunk 1 above)
Comment (DanielRuf) [reply package.json]: Hm, thought we use snyp or something similar. Will do it when I come to the installation of yarn.

File: test/javascript/index.html (hunk 2 above)
Comment (ncoden) [reply test/javascript/index.html]: hmmm I did not seen that, sorry.

File: package.json (hunk 1 above)
Comment (ncoden) [reply package.json]: I took care of that.

**Review by ncoden — APPROVED** [review]
👍 LGTM, thank you @DanielRuf

**Comment by ncoden** [comment]
I guess that tests fail here because of the focus bug you described at #11177 ?

**Comment by DanielRuf** [comment]
> I guess that tests fail here because of the focus bug you described at #11177 ?

Exactly.

**Comment by ncoden** [comment]
Ok. I'll take a look at this bug later. Like, why does this happens only now with updated test dependencies ?

**Comment by DanielRuf** [comment]
> Ok. I'll take a look at this bug later. Like, why does this happens only now with updated test dependencies ?

It also happened before but the new dependencies clearly show the issue instead of hiding it. Before this we saw it only with the previous Browserstack setup and now the testsuite fails due to this (before this it got a timeout with an undefined exit code).

**Pull request closed without merging**